File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.js
function solution(babbling) {
    let regex = /aya|ye|woo|ma/g;
    let answer = 0;
    for (i in babbling) {
        let test = babbling[i].replace(regex, "");
        if (test === "") {
            answer += 1;
        }
    }
    return answer;
}

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.swift
import Foundation

func solution(_ babbling:[String]) -> Int {
   var answer: Int = 0
    for babble in babbling {
        var newString: String
        if babble.contains("ayaaya") { continue }
        else if babble.contains("yeye") { continue }
        else if babble.contains("woowoo") { continue }
        else if babble.contains("mama") { continue }
        else {
            newString = babble.replacingOccurrences(of: "aya", with: ".")
            newString = newString.replacingOccurrences(of: "ye", with: ".")
            newString = newString.replacingOccurrences(of: "woo", with: ".")
            newString = newString.replacingOccurrences(of: "ma", with: ".")
            if newString.filter { $0 == "." }.count == newString.count {
                answer += 1
            }
        }
    }
    return answer
}